dmutils/email.py

```diff
@@ -231,3 +232,95 @@ def decode_invitation_token(encoded_token):
 
 def token_created_before_password_last_changed(token_timestamp, user_timestamp):
     return token_timestamp < user_timestamp
+
+
+class DMNotifyClient(object):
+    """Digital Marketplace wrapper around the Notify python client."""
+
+    client_class = NotificationsAPIClient
+
+    def __init__(self, api_key, base_url='https://api.notifications.service.gov.uk', logger=None):
+        """Set up logging and mail client."""
+        self.logger = logger or current_app.logger
+        self.client = self.client_class(api_key, base_url)
+
+    @staticmethod
+    def get_reference(email_address, template_id, personalisation, time=None, set_time=True, **kwargs):
+        """
+        Method to return the standard reference given the variables the email is sent with.
+
+        :param email_address: Emails recipient
+        :param template_id: Emails template ID on Notify
+        :param personalisation: Template parameters
+        :param time: Generally not wise to set this. Spoofs time that the email was sent.
```

Comment:
Very true, I was thinking if we want to do this for some sort of ordering or something but it's not necessary yet (or possibly ever). Removed.

As an aside it could still be overwritten by passing in a `time` kwarg